Makefile

```diff
@@ -0,0 +1,11 @@
+run:
+	 python dojopuzzles/manage.py runserver
+
+migrate:
+	python dojopuzzles/manage.py migrate
+
+update-static-files:
+	sudo rm -rf  staticfiles/* && python manage.py collectstatic
```

Comment:
**question**: Why use `sudo`? 

**suggestion**: Use [--clear](https://docs.djangoproject.com/en/3.1/ref/contrib/staticfiles/#cmdoption-collectstatic-clear) to clear old static files that are not used anymore.